src/generate.js

```diff
@@ -1,6 +1,7 @@
 import { readFileSync, writeFileSync } from 'fs';
+import { printTypes } from './type-printer';
```

Comment:
`type-printer` filename can be changed to `printTypes` to match the function name